packages/peregrine/lib/Price/price.js

```diff
@@ -38,10 +38,10 @@ export default class Price extends PureComponent {
     };
 
     render() {
-        const { value, currencyCode, classes } = this.props;
+        const { value, currencyCode, classes, locale } = this.props;
```

Comment:
You need to add locale to `propTypes` above.

It should be required, but that'd be a breaking change due to the way propTypes works. So I think we has it as optional and default it to en-US if not supplied? What do you think?

Reply:
If it can stay optional, it probably can be `undefined` as before when not provided.

Reply:
It's interesting that `locale` is a prop at all, though.

What was the reasoning behind making `locale` a prop here and calling `useIntl()` in every component that renders `Price`? As opposed to just calling `useIntl()` once within `Price`.